README.md

```diff
@@ -65,3 +65,52 @@ noirup --path ./git/noir
 **Tip**: All flags have a single character shorthand equivalent! You can use `-v` instead of `--version`, etc.
 
 ---
+
+## Github Action
+
+This action is in early development and so will likely experience breaking changes. It's recommended to pin to a particular tagged version.
+
+---
+
+Noirup is also available as a GitHub action to allow easy installation of the Noir toolchain using noirup.
+
+<br>
+
+## Example workflow
+
+```yaml
+name: test suite
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: nargo test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: noir-lang/noirup@0.1.0
```

Comment:
```suggestion
      - uses: noir-lang/noirup@v0.1.0
```
I think we need the `v` because GHA just grabs the tag verbatim.

Reply:
Good catch, thanks.